src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  max-width: $max-width;
+  padding: 20px 10px;
+  gap: 1px;
+
+  &__day {
+    display: block;
+    box-sizing: border-box;
+    height: $standart-size;
+    width: $standart-size;
+    border: 1px solid #000;
+    background-image: linear-gradient(to bottom,
+    rgba(156, 154, 154, 0.7), #fff, rgb(156, 156, 156, 0.9));
+
+    font-style: arial, sans-serif;
+    font-size: 30px;
+    line-height: $standart-size;
+    text-align: center;
+
```

Comment:
add transition properties such as background and transform to specify what exactly you'd like to change

Reply:
Could you please explain me this comment? I don't understand what exactly should I add.